wazo_acceptance/assets/wazo-service-test.py

```diff
@@ -90,21 +99,21 @@ def _are_all_items_matched_in_buffer(buf, matcher, items):
 
 
 def main():
-    assert _is_monit_running()
+    assert _is_monit_running(expected=True)
     assert _are_services_running(BASE_SERVICES)
 
     _wazo_service('stop')
-    assert not _is_monit_running()
+    assert not _is_monit_running(expected=False)
```

Comment:
Hard to read
What do you think about having 2 functions:
`assert _monit_is_running()`
`assert _monit_is_not_running()`

Reply:
Ooooh yes, very good idea, thanks :)